dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureRepository.java

```diff
@@ -118,6 +121,16 @@ public FeatureRepository(WsResource res, BundleContext bundleContext) {
         this.bundleContext = bundleContext;
     }
 
+    /**
+     * Use to check if a feature name is a commonly used alternate to an existing feature name
+     *
+     * @param featureName
+     * @return The existing feature name or null if no match
+     */
+    public String matchesAlternate(String featureName) {
+        return alternateFeatureNameToPublicName.get(featureName);
```

Comment:
We likely need to ignore case here like we do for short names:

`return alternateFeatureNameToPublicName.get(lowerFeature(featureName));`